link-grammar/connectors.c

```diff
@@ -424,7 +424,7 @@ static bool sort_condesc_by_uc_constring(Dictionary dict)
 		condesc_t *condesc = dict->contable.hdesc[n].desc;
 
 		if (NULL == condesc) continue;
-		calculate_connector_info(condesc);
+		calculate_connector_info(&dict->contable.hdesc[n]);
 		sdesc[i++] = dict->contable.hdesc[n].desc;
```

Comment:
this seems to be same as `condesc`

Reply:
`condesc` is `dict->contable.hdesc[n].desc`, not `dict->contable.hdesc[n]`.
To be clearer,  these lines can be changed to (not tested):
```
hdesc_t *hdesc = &dict->contable.hdesc[n];

if (NULL == hdesc->desc) continue;
calculate_connector_info(hdesc);
sdesc[i++] = hdesc->desc;
```